src/managednetworkfabric/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+6.1.0
```

Comment:
Backward incompatible changes should increase version in major part, which is 7.0.0
please refer to extension versioning doc here: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md

Reply:
We are wanting to keep the az CLI release version in correspondence to the latest release version of our service managed network fabric